server/src/bots/telegram/botResponse/startResponse.ts

```diff
@@ -49,13 +46,14 @@ export const startResponse: CallBackQueryHandlerWithCommandArgument = async (
                 chatId
             );
         } else {
-            logger.log(
+            logger.error(
                 'info',
-                `New user ${newUser.chatId} was successfully added`,
+                new Error(`New user ${newUser.chatId} was successfully added`),
```

Comment:
I think there is no need to create error,
You could simply write ```logger.log('error', ``New user ${newUser.chatId} was successfully added`, LogCategory.Command, chatId);```

Reply:
It still logs as error.
```logger.error()``` method might be useful when we are logging some exceptions happened in system and were caught.